client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -63,7 +66,8 @@ const designSetup: Step = function DesignSetup( { navigation } ) {
 		site?.launch_status === 'unlaunched' && site?.options?.is_automated_transfer
 	);
 
-	const showDesignPickerCategories = isEnabled( 'signup/design-picker-categories' );
+	const showDesignPickerCategories =
+		isEnabled( 'signup/design-picker-categories' ) && ! isAnchorSite;
```

Comment:
Not strictly related to this PR, but it looks like maybe we lost the logic from [this PR](https://github.com/Automattic/wp-calypso/pull/61964) during the Stepper transition? Or am I misreading?

Reply:
You're right! Great catch. I started an issue for this here: https://github.com/Automattic/wp-calypso/issues/62779